styles/base.css

```diff
@@ -449,3 +529,79 @@ header {
         left: 0 !important;
     }
 }
+
+/* Blog */
+
+@media (max-width: 992px) {
+    .blog.row {
+        --bs-gutter-x: 0;
+    }
+}
+
+.post-preview {
+    margin-top: calc(80rem / 16);
+}
+
+.post-preview:first-of-type {
+    margin-top: calc(48rem / 16);
+}
```

Comment:
The first post's margin-top on Mobile is shorter than the margin-top of the other posts. On Desktop, the margin-top on all posts is the same.